string action
string object_name
bool allow_repeat
---
bool completed